src/rev.md

```diff
@@ -19,58 +19,63 @@ Both the inplace and "normal" variant return the gradient. The difference is tha
 ### Usage story
 Let us start by looking at the most basic examples we can think of:
 
-$f(x,y) = x^2 + y*3$
+\\[ f(x,y) = x^2 + 3y \\]
 
-We have two input variables $x$, $y$ and a scalar return value.  Just to check our sanity, we first pass it to [wolfram alpha](https://www.wolframalpha.com/input?i2d=true&i=D%5BPower%5Bx%2C2%5D+%2B+y*3%2Cx%5D%3B+D%5BPower%5Bx%2C2%5D+%2B+y*3%2Cy%5D%3B+). No big surprises so far. Let's check for Enzyme (our compiler explorer does not handle Rust yet, so you'l have to trust me on this).
+We have two input variables \\(x\\), \\(y\\) and a scalar return value.  Just to check our sanity, we first pass it to [wolfram alpha](https://www.wolframalpha.com/input?i2d=true&i=D%5BPower%5Bx%2C2%5D+%2B+y*3%2Cx%5D%3B+D%5BPower%5Bx%2C2%5D+%2B+y*3%2Cy%5D%3B+). No big surprises so far. Let's check for Enzyme (our compiler explorer does not handle Rust yet, so you'l have to trust me on this).
 
 ```rust
 #[autodiff(df, Reverse, Active, Active, Active)]
 fn f(x: f32, y: f32) -> f32 {
-  x*x + 3.0 * y
+  x * x + 3.0 * y
 }
 ```
 
 Enzyme actually generates the code on LLVM-IR level, but Rust is nicer to read, so I will pretend we would generate a Rust implementation:
 
 ```rust
 fn f(x: f32, y: f32) -> f32 {
-  x*x + 3.0 * y
+  x * x + 3.0 * y
 }
 fn df(x: f32, y: f32) -> (f32, f32, f32) {
-  (2.0 * x, 3.0, x*x + 3.0 * y)
+  let d_dx = 2.0 * x;
+  let d_dy = 3.0;
+  let f = x * x + 3.0 * y;
+  (d_dx, d_dy, f)
```

Comment:
I think it's more clear to name these variables rather than explain a big tuple in text.

I personally think of `dx` as being a variation in `x`, which is correct for forward mode. In reverse mode, the notation is usually $\bar x$, which is shorthand for $\partial (\cdot)/\partial x$ where the functional $(\cdot)$ is implicit and assumed to be clear from context. I'm happy to use other naming here, but I think using `dx` for this quite different object is confusing.